### PR TITLE
made Client Side with npx create-react

### PR DESCRIPTION
**Pull request by revengemiroz**
made a running react app

### CODE REVIEW

File: client/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```
Comment (ivanms1) [inline client/README.md]: can you remove this? we will use the readme in the other folder

File: client/src/serviceWorker.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
+// This optional code is used to register a service worker.
```
Comment (ivanms1) [inline client/src/serviceWorker.js]: can you remove this and its imports?

File: client/README.md (hunk 1 above)
Comment (revengemiroz) [reply client/README.md]: done

File: client/src/serviceWorker.js (hunk 2 above)
Comment (revengemiroz) [reply client/src/serviceWorker.js]: done

**Review by ivanms1 — APPROVED** [review]

**Pull request merged**